flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/operators/over/BufferDataOverWindowOperatorTest.java

```diff
@@ -237,4 +241,9 @@ public StreamingRuntimeContext getRuntimeContext() {
 	private void addRow(Object... fields) throws Exception {
 		operator.processElement(new StreamRecord<>(GenericRow.of(fields)));
 	}
+
+	private interface EnvironmentSupport {
```

Comment:
this seems never be used.

Reply:
Yes, leftover from the previous version. Thanks.